opensuse-kubectl-docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#Base image for kube yaml installer
+FROM opensuse:42.2
+
+#install: curl, kubectl, unzip, openssh, sshpass, vim, nano
+RUN true \
+    && zypper ref \
+    && zypper --non-interactive in unzip \
+    openssh \
+    sshpass \
+    vim \
+    nano \
```

Comment:
no, but when I was creating this image for the first time, people requested all this on our channel, so I added it. The idea is to use this image for other stuff as well and not to create opensuse images for specific needs